pkg/cli/admin/release/mirror.go

```diff
@@ -464,6 +471,26 @@ func (o *MirrorOptions) Run() error {
 	if is == nil {
 		o.ImageStream = &imagev1.ImageStream{}
 		is = o.ImageStream
+
+		// get release architecture
+		infoBuf := &bytes.Buffer{}
+		infoOpts := NewInfoOptions(genericclioptions.IOStreams{Out: infoBuf, ErrOut: o.ErrOut})
```

Comment:
I think we can drop this hunk and the `info.go` changes.  We should be able to grab the arch in `ImageMetadataCallback` below from [`config.Architecture`][1], with something like:

```go
extractOpts.ImageMetadataCallback = func(m *extract.Mapping, dgst, contentDigest digest.Digest, config *dockerv1client.DockerImageConfig) {
  releaseDigest = contentDigest.String()
  verifier.Verify(dgst, contentDigest)
  if config != nil && config.Architecture != "" {
    if val, ok := archMap[config.Architecture]; ok {
      archExt = "-" + val
    } else {
      archExt = "=" + config.Architecture
    }
  }
}
```

[1]: https://github.com/openshift/oc/blob/0ccbf0633fd374af88f444d9e5cc6bd595933670/vendor/github.com/openshift/library-go/pkg/image/dockerv1client/types.go#L91